pom.xml

```diff
@@ -45,7 +45,7 @@
         <jdk.version>1.8</jdk.version>
         <maven-compiler-plugin.version>2.0.2</maven-compiler-plugin.version>
         <maven-jar-plugin.version>2.2</maven-jar-plugin.version>
-        <junit.version>4.7</junit.version>
+        <junit.version>4.12</junit.version>
```

Comment:
Tried using ClassRule instead of Rule which is not available in 4.7. After that removed ClassRule, but kept the version upgrade. Do you see any problems with 4.12? @yogeshnachnani